lit/docs/config/basic-schemas.lit

```diff
@@ -61,22 +61,30 @@ just here for completeness!
 }
 
 \schema{identifier}{
-  An \italic{identifier} is string value. Currently, there are no validations
-  for what makes a string a valid identifier.
+  An \italic{identifier} is string value. Currently, the validation will only
+  show as warnings. For the sake of future-proofing, you may want to conform to it.
```

Comment:
Could you move this comment about validation to the end (after you describe what the validation is)?

So, it'd be something like:

> An \italic{identifier} is a string value. The following defines the allowed character set for an \italic{identifier}:
> 
> ...
> 
> Currently, the validation will only show as warnings. For the sake of future-proofing, you may want to conform to it.

Reply:
Also, I know this was a problem before, but I just noticed the first sentence should be "An \italic{identifier} is **a** string value" (was missing the "a" before).